### DISCUSSION TITLE
Area not changing

### DISCUSSION THREAD

**Discussion by pikmine1** [OpenTabletDriver/OpenTabletDriver#2429] [Q&A] [answered]
Hi, I just got a XP-PEN Star 640 tablet and when I try to change the area of the tablet it just doesn't work, it stays the same. I have tried uninstalling the devices multiple times and nothing changed, it's still full area all the time. (Yes I pressed save and apply...)

Here is a screenshot of the console if anyone could help me. Thanks a lot

![image_2022-09-19_213759048](https://user-images.githubusercontent.com/20427808/191148292-9093b836-6c9a-405b-859e-88286fa3e58d.png)

**Comment by jamesbt365** [comment, marked as answer]
Uninstall your other driver

**Reply by pikmine1** [reply]
I have tried through their app and manually yet I have not succeeded, can you like link a vid to help me or just tell me where to look because i've looked and I can't find anything

**Reply by pikmine1** [reply]
I uninstalled any driver named "USB driver" that was either hidden or active except my mouse. I then restarted my computer, launched OpenTabletDriver and plugged in my tablet, still doesnt apply a new area. Here is the console this time:

[Detect:Info]	Searching for tablets...
[Detect:Info]	No tablets were detected.
[Settings:Info]	Presets have been refreshed. Loaded 0 presets.
[Detect:Info]	Searching for tablets...
[Detect:Info]	Searching for tablets...
[Device:Warning]	Failed to set device output: 02-B0-02
[Device:Warning]	Failed to set device output: 02-B0-02
[Detect:Info]	Found tablet 'XP-Pen Star G640'
[Detect:Info]	Found tablet 'XP-Pen Star G640'
[XP-Pen Star G640:Info]	Display area: [1920x1080@<960, 540>:0°],
[XP-Pen Star G640:Info]	Tablet area: [45x25.3125@<80, 50>:0°],
[XP-Pen Star G640:Info]	Clipping: Enabled
[XP-Pen Star G640:Info]	Ignoring reports outside area: Disabled
[XP-Pen Star G640:Info]	Tip Binding: [Mouse Button Binding: Left]@0%
[Settings:Info]	Driver is enabled.
[Settings:Info]	Presets have been refreshed. Loaded 0 presets.
[Settings:Info]	Presets have been refreshed. Loaded 0 presets.

**Reply by jamesbt365** [reply]
The failed to set device output means that its already initialized but another driver, this means you need to uninstall that other driver, these are typically accessible through apps and features. You'll find something named pentablet. If you do not see it here then install the xp-pen tablet driver again and then uninstall it this way.

Deleting files is not uninstalling.

**Reply by pikmine1** [reply]
I have done this once before but I can try again, downloading the Pentablet app and drivers and uninstalling them through "Add or remove a program". Ill do this now and update you.

**Reply by jamesbt365** [reply]
Either way this is not a problem with OpenTabletDriver. It simply cannot access the device because it is currently in use by another driver. You may be able to find something by looking at the running services as well, but this is the right answer.

**Reply by pikmine1** [reply]
That really doesn't help me, I have just done everything including removing my firewall like the program for Pentablet said. I have installed it, restarted my computer, didn`t work. So I uninstalled it, restarted my computer, didn`t work. You saying this is the right answer isn`t helping me. I have uninstalled countless drivers and all the advice im getting is "uninstall more". If you could help me with more it would be greatly appreciated.

**Reply by jamesbt365** [reply]
Marking this unanswered doesn't change anything, this is a piece of software on **your** computer, stopping **our** software from functioning as intended, it's not **our** responsibility to find the piece of software on **your** computer that is causing this, this is another **tablet driver** initing the tablet and using it before we can, i recommend you explore the device container in device manager, but stop unmarking this as answered. This is the problem, i have answered with the correct answer. Its a problem with your software on your system.

**Reply by pikmine1** [reply]
And that's exactly what I need help with and it's not what i'm given. I have disabled every device there is that were related with peripherals and some that were not. Everytime nothing helped. You telling me this is the answer isn't helping me I'm trying to get your software to work but you seem to be stuck on what is an answer or not. For me I think this is the correct answer since you're the one with experience, but you're not telling me how to solve the issue with the answer or anything. In the past minutes I have disabled more device in device manager to the point my keyboard wasn't working properly, my mouse was disabled a few times and my audio stopped working for a bit. Now if you could help me identify what MY issue is I'd be glad to continue this conversation as I need help, not an answer. He is a screenshot of my devices.
![image_2022-09-20_191729382](https://user-images.githubusercontent.com/20427808/191381215-f7748df4-e3ac-483f-aed1-111355453e47.png)

**Reply by pikmine1** [reply]
Alright I have solved my issue, it was a physical peripheral issue, my tablet was plugged in a USB splitter and that was interfering with it. I'm sorry for bothering you and thank you for your kind work in this community. I appologize if I came off as rude that was not the intention, anyways I hope you continue helping dumb people like me. Also for future people that ask for help and have this specific issue I guess you could recommend them unplugging any USB splitter. Thanks again.

**Reply by pikmine1** [reply]
Oh yeah you can mark this as solved

**Reply by lUIS221122** [reply]
> Oh sim, você pode marcar isso como resolvido



> Tudo bem que resolvi meu problema, foi um problema periférico físico, meu tablet estava conectado em um divisor USB e que estava interferindo com ele. Sinto muito por incomodá-lo e obrigado por seu trabalho na comunidade. Eu appologize se eu saí como rude que não era a intenção, de qualquer maneira eu espero que você continue ajudando pessoas como eu. Também para futuras pessoas que pedem ajuda e têm esse problema específico eu acho que você poderia recomendá-los desconectando qualquer divisor USB. Obrigado de novo.

OII estou com o mesmo problema se puder me ajudar to triste queria usar o drive
